config/environments/production.rb

```diff
@@ -46,6 +46,7 @@
 
   config.force_ssl = true
   config.ssl_options = {
+    hsts: { preload: ENV['HSTS_PRELOAD'] == 'true' ? true : false },
```

Comment:
According to the [doc](https://api.rubyonrails.org/v6.1.7/classes/ActionDispatch/SSL.html) and the [source code](https://github.com/rails/rails/blob/v6.1.7/actionpack/lib/action_dispatch/middleware/ssl.rb#L58) the default is `false` yes :

> preload: Advertise that this site may be included in browsers' preloaded HSTS lists. HSTS protects your site on every visit except the first visit since it hasn't seen your HSTS header yet. To close this gap, browser vendors include a baked-in list of HSTS-enabled sites. Go to [hstspreload.org](https://hstspreload.org/) to submit your site for inclusion. Defaults to false.